src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3774,6 +3774,93 @@ public void GetPartCallsGetPartCore()
             // Validate we get the same object back if we call GetPart again
             Assert.Same(part, mockPackage.GetPart(partUri));
         }
+        
+        [Fact]
+        public void ComparePackUriSamePackSamePart()
+        {
+            Uri partUri = new Uri("/idontexist.xml", UriKind.Relative);
+            Uri packageUri = new Uri("application://");
+
+            Uri combinedUri = PackUriHelper.Create(packageUri, partUri);
+            Assert.Equal(combinedUri.ToString(), "pack://application:,,,/idontexist.xml");
+
+            Uri sameCombinedUri = PackUriHelper.Create(packageUri, partUri);
+            Assert.Equal(combinedUri.ToString(), "pack://application:,,,/idontexist.xml");
```

Comment:
Nit: The arguments here and to some of the other asserts should be reversed. The first argument is "expected", the second is "actual".